style.css

```diff
@@ -0,0 +1,67 @@
+body {
+  background-color: #eee;
+  font-family: sans-serif;

```

Comment:
Please use a different font-family. There is so much nice looking default fonts you can find and your website will look much better! Thanks 😄